c2corg_ui/externs/bootstrap-slider.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @fileoverview Externs for bootstrap-slider
+ *
+ * @externs
+ */
+
+
+/**
+ * @constructor
+ */
+function Slider(arg1, arg2) {};
+
+
+/**
+ * @param {Object} arg
```

Comment:
Sorry,  @asaunier I missed your question. Indeed, as said by @tsauerwein, we should only type the options we actually use.